operations/finance/risk-management/mattermost-templated-agreements.md

```diff
@@ -6,18 +6,18 @@ description: >-
 
 # Mattermost U.S. Consulting Agreements
 
-This agreement is typically used for vendors who do not require Mattermost to sign a [custom agreement](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-purchase/how-to-procure-a-vendor-contract#procuring-a-vendor). Before you kick off the process, please confirm your vendor satisfies the criteria below. If you are not sure, please reach out to [People Ops](https://community.mattermost.com/private-core/channels/ask-people-team).
+This agreement is typically used for vendors who do not require Mattermost to sign a [custom agreement](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-purchase/how-to-procure-a-vendor-contract#procuring-a-vendor). Before you kick off the process, please confirm your vendor satisfies the criteria below. If your vendor does not satisfy the criteria below, this vendor contract should follow the [Procurement Process](https://handbook.mattermost.com/operations/finance/purchasing/how-to-procure-a-vendor-contract). If you are not sure, please reach out to [People Ops](https://community.mattermost.com/private-core/channels/ask-people-team).
 
 ## **Criteria:**
 
-* A firm or an individual with an LLC 
+* An individual with an LLC 
 * Provide ad-hoc services to Mattermost, examples include:  
-  * Project-Based: for a defined period with pre-set project milestones
-  * Ad-hoc services: work that is \(i\) not continuous \(e.g. hours vary month to month\) and \(ii\) typically assigned on an as-needed basis
-
+  * On-going nature: the engagement will exceed 6 months and does not have a fixed end date, e.g. on-going budget approval will not be needed. 
```

Comment:
- We do not typically and should not be procuring consulting services for "ad-hoc" purposes. All procurement of services should have a statement of work detailing the services to be 
- Is there are reason this is time floored at 6 months?
- The contracts should also have a term on them, this is how the original budget is determined through procurement.